test/e2e/specs/specs-playwright/wp-invite__revoke.ts

```diff
@@ -52,7 +52,8 @@ describe( DataHelper.createSuiteTitle( `Invite: Revoke` ), function () {
 	it( 'Revoke the invite for test user', async function () {
 		await sidebarComponent.navigate( 'Users', 'All Users' );
 		await peoplePage.clickTab( 'Invites' );
-		await peoplePage.revokeInvite( testEmailAddress );
+		await peoplePage.selectInvitedUser( testEmailAddress );
+		await peoplePage.revokeInvite();
```

Comment:
Separated out the action to revoke invite and clicking on the pending invite into separate methods, so the actions are not coupled to one another.